source/blog/2016-12-06-part1-install-origin-on-f25-atomic-host.html.md

```diff
@@ -0,0 +1,357 @@
+we'll take the system we just installed for a spin and make sure
+everything is working as expected.
+
+Environment
+===========
+
+I've tried to make this setup as generic as possible. In this case I
+will be targeting three generic servers that are running Fedora 25
+Atomic Host. As is common with cloud environments these servers each
+have an "internal" private address that can't be accessed from the
+internet, as well as a public NATed address that can be accessed from
+the outside. Here is the identifying information for the three servers:
+
+|    Role     |  Public IPv4   | Private IPv4 |
+|-------------|----------------|--------------|
+| master,etcd | 54.175.0.44    | 10.0.173.101 |
+| worker      | 52.91.115.81   | 10.0.156.20  |
+| worker      | 54.204.208.138 | 10.0.251.101 |
+
+As you can see from the table I've marked one of the nodes as the master
+and the other two as what I'm calling *worker nodes*. The master node
+will run the api server, scheduler and, controller manager. We'll also
+run etcd on it. Since we want to make sure we don't starve the node running
+etcd, we'll mark the master node as **unschedulable** so that
+application containers don't get scheduled to run on it.
+
+The other two nodes, the worker nodes, will have the proxy and the
+kubelet running on them; this is where the containers (inside of pods)
+will get scheduled to run. We'll also tell the installer to run a
+registry and an HAProxy router on the two worker nodes so that we can
+perform builds as well as access our services from the outside world via
+HAProxy.
+
+The Installer
+=============
+
+[Openshift Origin](https://github.com/openshift/origin) uses
+[Ansible](https://www.ansible.com/) to manage the installation of
+different nodes in a cluster. The code for this is aggregated in the
+[OpenShift Ansible
+Installer](https://github.com/openshift/openshift-ansible) on GitHub.
+Additionally, to run the installer you'll also need to
+[install ansible](http://docs.ansible.com/ansible/intro_installation.html#installing-the-control-machine).
+
+Now we can start by cloning the repo:
+
+    $ git clone https://github.com/openshift/openshift-ansible.git &>/dev/null
+    $ cd openshift-ansible/
+    $ git checkout 6d2a27240b982822657e6ee463e2e66ce8a3a770
+    Previous HEAD position was 6d2a272... Merge pull request #2884 from sdodson/image-stream-sync
+    HEAD is now at 734b9ae... Merge pull request #2876 from dustymabe/dusty-fix-etcd-selinux
+
+In order to document this better in this blog post we are specifically
+checking out commit `6d2a27240b982822657e6ee463e2e66ce8a3a770` so that
+others can reproduce. You shouldn't have to do this but if you hit a bug
+in the lastest master please open an issue upstream!
+
+Now that we have the installer we created an inventory file called
+`myinventory` in the same directory as the git repo. Your inventory file
+can be anywhere, but we chose to place it here.
```

Comment:
"The inventory file can be anywhere, but for this install we'll place it there."